nautobot_golden_config/models.py

```diff
@@ -15,10 +16,22 @@
 from nautobot.extras.utils import extras_features
 from nautobot.utilities.utils import serialize_object, serialize_object_v2
 from netutils.config.compliance import feature_compliance
-from nautobot_golden_config.choices import ComplianceRuleConfigTypeChoice
+from nautobot_golden_config.choices import ComplianceRuleConfigTypeChoice, RemediationTypeChoice
 from nautobot_golden_config.utilities.constant import ENABLE_SOTAGG, PLUGIN_CFG
 from nautobot_golden_config.utilities.utils import get_platform
 
+# temporal implementation until MAPPERS operational in netutils, and netutils > 1.4.0
```

Comment:
Let's just bump the netutils verison

Reply:
it is also associated with the nautobot version, current dependency (1.5.3) requires netutils = "~1.3.0". We would need to bump both.

Reply:
hmm, I thought we could be more restrictive

Reply:
What @jeffkala communicated yesterday was we will target remediation for Nautobot ~1.5.13.